[BUG] glib support does not build on older versions anymore, because of `G_FORMAT_SIZE_BITS`

When compiling with GLib 2.48, the build fails with
```
vendor/github.com/gotk3/gotk3/glib/glib.go:497:44: could not determine kind of name for C.G_FORMAT_SIZE_BITS
```

I can't tell exactly when `G_FORMAT_SIZE_BITS` was added to GLib but it seems to be after 2.48. This reference and the others for `FormatSizeFlags` seems to have been introduced in https://github.com/gotk3/gotk3/commit/cf79ce840c3f145389ab59e135f5248cc85fdfcb

--- a/glib/glib.go
+++ b/glib/glib.go
@@ -494,7 +494,6 @@ const (
 	FORMAT_SIZE_DEFAULT     FormatSizeFlags = C.G_FORMAT_SIZE_DEFAULT
 	FORMAT_SIZE_LONG_FORMAT FormatSizeFlags = C.G_FORMAT_SIZE_LONG_FORMAT
 	FORMAT_SIZE_IEC_UNITS   FormatSizeFlags = C.G_FORMAT_SIZE_IEC_UNITS
-	FORMAT_SIZE_BITS        FormatSizeFlags = C.G_FORMAT_SIZE_BITS
 )
 
 // FormatSizeFull is a wrapper around g_format_size_full().

--- a/glib/glib_since_2_58.go
+++ b/glib/glib_since_2_58.go
@@ -1,0 +1,16 @@
+// Same copyright and license as the rest of the files in this project
+
+// +build !glib_2_40,!glib_2_42,!glib_2_44,!glib_2_46,!glib_2_48,!glib_2_50,!glib_2_52,!glib_2_54,!glib_2_56
+
+package glib
+
+// // #include <gio/gio.h>
+// // #include <glib.h>
+// // #include <glib-object.h>
+// // #include "glib.go.h"
+// // #include "glib_since_2_44.go.h"
+// import "C"
+
+const (
+	FORMAT_SIZE_BITS FormatSizeFlags = C.G_FORMAT_SIZE_BITS
+)

